backend/src/main/java/me/performancereservation/domain/admin/service/AdminReservationService.java

```diff
@@ -0,0 +1,66 @@
+package me.performancereservation.domain.admin.service;
+
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.domain.admin.dto.AdminReservationPageResponse;
+import me.performancereservation.domain.admin.repository.AdminReservationRepository;
+import me.performancereservation.domain.reservation.Reservation;
+import me.performancereservation.domain.reservation.ReservationRepository;
+import me.performancereservation.domain.reservation.enums.ReservationStatus;
+import me.performancereservation.global.exception.ErrorCode;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+@Service
+@RequiredArgsConstructor
+public class AdminReservationService {
+    private final ReservationRepository reservationRepository;
+    private final AdminReservationRepository adminReservationRepository;
+
+    /** 관리자 예약 목록 조회
+     *
+     * @param pageable
+     * @return AdminReservationPageResponse
+     */
+    @Transactional(readOnly = true)
+    public Page<AdminReservationPageResponse> getReservationList(Pageable pageable) {
+        return adminReservationRepository.findAdminReservations(pageable);
+    }
+
+    /** 관리자 예약 목록 검색
+     *
+     * @param pageable 페이징
+     * @param userName 회원 이름(검색 키)
+     * @param performanceTitle 공연 제목(검색 키)
+     * @param reservationStatus 예약 상태(검색 키)
+     * @param startDate 시작 날짜(검색 키)
+     * @param endDate 종료 날짜(검색 키)
+     * @return AdminReservationPageResponse
+     */
+    @Transactional(readOnly = true)
+    public Page<AdminReservationPageResponse> searchReservationList(Pageable pageable,
+                                                                    String userName,
+                                                                    String performanceTitle,
+                                                                    ReservationStatus reservationStatus,
+                                                                    LocalDateTime startDate,
+                                                                    LocalDateTime endDate) {
+        return adminReservationRepository.searchAdminReservations(userName, performanceTitle, reservationStatus, startDate, endDate, pageable);
+    }
+
+    /** 관리자 예약 확정 상태 변경
+     *
+     * 예약을 확정 상태로 변경
+     * @param reservationId 예약 id
+     */
+    @Transactional
+    public void confirmReservation(Long reservationId) {
+        Reservation reservation = reservationRepository.findById(reservationId)
+                .orElseThrow(() -> ErrorCode.RESERVATION_NOT_FOUND.domainException("예약이 존재하지 않습니다."));
+
+        reservation.confirm();
+    }
```

Comment:
```suggestion
    /** 관리자 예약 확정 상태 변경
     *
     * 예약을 확정 상태로 변경
     * @param reservationId 예약 id
     */
    @Transactional
    public void confirmReservation(Long reservationId) {
        Reservation reservation = reservationRepository.findById(reservationId)
                .orElseThrow(() -> ErrorCode.RESERVATION_NOT_FOUND.domainException("예약이 존재하지 않습니다."));

        reservation.confirm();
        
        // TODO reservationId로 PerformanceSchedule 엔티티 조회
        // TODO schedule의 remainingSeats를 reservation의 quantity(티켓 수량) 만큼 차감
    }
```

레디스로 좌석을 선점하도록 예약을 구현해둔 상태라서 

reservation에 대한 공연 회차의 좌석을 차감 처리를 해줘야 데이터 정합성이 유지됩니다!

Reply:
잘부탁드립니다 (_ _)

Reply:
앗 남은 좌석 처리 생각을 못했네요. 그럼 처리해야 할 부분이 회차 엔티티의 남은 좌석 수만 차감 시키면 되는 건가요? 
레디스에 대한 추가 처리 할 부분이 또 있을까요?

Reply:
레디스는 이미 선점하는 과정속에서 차감되어있는 상태라 RDB 테이블의 정합성만 맞춰주면 됩니다!